omas/tests/test_omas_utils.py

```diff
@@ -118,22 +125,28 @@ def test_omas_info(self):
             assert isinstance(ods_info_pfa['pf_active.circuit.0.connections.documentation'], basestring)
         ods_info_list = omas_info(get_list)
         assert all([item in ods_info_list for item in get_list])
+        return
 
     def test_o2u(self):
         assert o2u('equilibrium.time_slice.0.global_quantities.ip') == 'equilibrium.time_slice.:.global_quantities.ip'
         assert o2u('equilibrium.time_slice.:.global_quantities.ip') == 'equilibrium.time_slice.:.global_quantities.ip'
         assert o2u('2') == ':'
         assert o2u('equilibrium') == 'equilibrium'
         assert o2u('equilibrium.2') == 'equilibrium.:'
+        return
 
     def test_set_time_array(self):
         ods = ODS()
         ods.set_time_array('equilibrium.vacuum_toroidal_field.b0', 0, 0.1)
         ods.set_time_array('equilibrium.vacuum_toroidal_field.b0', 1, 0.2)
+        return
 
     def test_info(self):
         omas_info('equilibrium')
         omas_info(None)
+        return
+
+    # End of TestOmasUtils class
```

Comment:
This also makes the class end easier to find, and makes it easier to fold (at least for me in pycharm) the whole class since the last line of the class is no longer the last line of a function.